domain/build.gradle.kts

```diff
@@ -3,7 +3,8 @@ plugins {
     id("org.jetbrains.kotlin.jvm")
 }
 java {
-    sourceCompatibility = JavaVersion.VERSION_1_7
-    targetCompatibility = JavaVersion.VERSION_1_7
+    sourceCompatibility = JavaVersion.VERSION_11
+    targetCompatibility = JavaVersion.VERSION_11
 }
+
```

Comment:
Couldn't find a KoltinOptions to use here....  I guess it's something we don't have to worry about until we see an issue. In other projects it hasn't been needed.